src/hotspot/cpu/aarch64/macroAssembler_aarch64.hpp

```diff
@@ -445,9 +445,9 @@ class MacroAssembler: public Assembler {
   inline void msub(Register Rd, Register Rn, Register Rm, Register Ra) {
     if (VM_Version::supports_a53mac() && Ra != zr)
       nop();
-    if (VM_Version::is_neoverse_n_series()) {
-      /* On Neoverse N series, MSUB uses the same ALU with SDIV.
-       * The combination of MUL/SUB can utilize multiple ALUS,
+    if (VM_Version::is_neoverse_family()) {
```

Comment:
Thanks for changing. However, currently neoverse_family includes N1,N2,V1 and V2 for which this change is ok but eventually when support is added for next versions of N/V series which do not require splitting `msub` into `mul` and `sub` (for ex. V3) then this check will have to be modified as V3 will need to be included in the "neoverse_family". Maybe a separate function here which checks for only those N/V series where this change will benefit.

Reply:
Let's not try to guess about the performance of future processors. `is_neoverse_family()` fine for now, because the performance issue affects all known Neoverse processors.